src/jit.rs

```diff
@@ -931,14 +970,16 @@ impl std::fmt::Debug for JitCompiler {
 
 impl JitCompiler {
     // Arguments are unused on windows
-    fn new<E: UserDefinedError>(_program: &[u8], _config: &Config) -> Result<Self, EbpfError<E>> {
+    fn new<E: UserDefinedError>(_program: &[u8], config: &Config) -> Result<Self, EbpfError<E>> {
```

Comment:
Unrelated changes, should come in their own PR.

Reply:
That's the spirit!

Reply:
Done. Since this was the only comment, I'm guessing this is now ready to land? 🥴